src/stories/list-reservation/ListReservationsAndLoans.tsx

```diff
@@ -0,0 +1,116 @@
+import { Counter, CounterProps } from "../counter/Counter";
+import { StatusLabel, StatusLabelProps } from "../status-label/StatusLabel";
+import { ArrowUI } from "../arrows/icon-arrow-ui/ArrowUI.stories";
+import { Material } from "../material/Material";
+
+export type ListReservationsAndLoansProps = {
+  material: {
+    type: string;
+    title: string;
+    description: string;
+    image: string;
+    url: string;
+    noteLabel: string;
+    noteUrl: string;
+    noteAsWarning: boolean;
+  };
+  isStacked: boolean;
+  counter: CounterProps;
+  statusLabel: StatusLabelProps;
+  statusNotes: string;
+  fee: number;
+};
+
+export const ListReservationsAndLoans = (
+  props: ListReservationsAndLoansProps
+) => {
+  const { material, isStacked, counter, statusLabel, statusNotes, fee } = props;
+
+  return (
+    <div className="list-container">
+      <div className="list-reservation">
+        <div className="list-reservation__material">
+          <div>
+            <Material size="small" animate={true} url={material.image} />
+          </div>
+          <div className="list-reservation__information">
+            <div>
+              <StatusLabel label="bog" status="outline" />
+            </div>
+            <div className="list-reservation__about">
+              <p className="text-header-h4">{material.title}</p>
```

Comment:
Is there a reason to use a `<p>` tag here for something that could be a headline?

Reply:
We have decided to use h3 as default, so I have changed it accordingly.